s3tests/functional/test_s3.py

```diff
@@ -7533,7 +7533,7 @@ def test_lifecycle_get():
 @attr(resource='bucket')
 @attr(method='put')
 @attr(operation='test lifecycle expiration')
-@attr('lifecycle')
+@attr('lifecycle_timing')
```

Comment:
maybe keep the `lifecycle` tag as it is, and add another one like `expiration` or `lifecycle_expiration`?